card/mod/basic_types/set/type/json.rb

```diff
@@ -1,5 +1,5 @@
 event :validate_json, :validate, on: :save, changed: :content do
-  check_json_syntax
+  check_json_syntax if content.present?
```

Comment:
ok for now.  wondering if we shouldn't have a default rule setting content to {}?

Reply:
I agree in principal but "" is only a bad query, "{}" searches for all cards. That could cause a lot of performance trouble.